core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -112,6 +112,21 @@
     public boolean contains(Scope scope) {
       return (mValue | scope.mValue) == mValue;
     }
+
+    /**
+     * @return the thrift representation of this scope field
```

Comment:
The convention is to put all wire types in the `wire` package. Now that `Scope` is being used as a wire type, we should either create a wire type copy, or move it to the `wire`